kpatch-build/create-diff-object.c

```diff
@@ -922,6 +1020,9 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				continue;
 			}
 
+#ifdef __powerpc__
+			add_off = 0;
+#else
```

Comment:
with ```-mcmodel=large```, the relative relocation or R_PPC64_REL24 types are limited to functions only. With -ffunction-sections gcc flag every function is the section and the section is replaced with their symbols by
```
if (rela->sym->sec && rela->sym->sec->sym) {
        rela->sym = rela->sym->sec->sym;
        continue;
}
```
for every data access, the -mcmodel=large generates two relocations types
```R_PPC64_TOC16_HA``` and ```R_PPC64_TOC16_LO_DS``` . Which is DS-Form load.
```add_off=0``` should be the right thing to do here.